packages/patterns/src/index.native.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import './store';
+import { lock } from './lock-unlock';
+
```

Comment:
@dcalhoun, @geriux, @fluiddot we have added a `patterns` package which will over time deprecate the `reusable-blocks` package. For native this is just exporting a null return component for `PatternsMenuItems` in the same way that the `[ReusableBlocksMenuItems](https://github.com/WordPress/gutenberg/blob/trunk/packages/reusable-blocks/src/components/reusable-blocks-menu-items/index.native.js)` does. When you have a moment can you just double-check that everything is still as it should be from the mobile app end, ie. no `Create pattern/reusable block` menu appears.

Reply:
From testing briefly, I did not encounter any unexpected changes. The UX appears to have remained the same in the mobile editor. Thank you for providing the opportunity to double-check! 🙇🏻